spring-cloud-gcp-starters/spring-cloud-gcp-starter-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/autoconfig/GcpPubsubAutoConfiguration.java

```diff
@@ -112,4 +117,16 @@ public PublisherFactory defaultPublisherFactory(GcpProjectIdProvider projectIdPr
 		return new DefaultPublisherFactory(projectIdProvider, subscriberProvider, channelProvider,
 				credentialsProvider);
 	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	public PubsubAdmin pubsubAdmin(GcpProjectIdProvider projectIdProvider) {
+		try {
+			return new PubsubAdmin(projectIdProvider, TopicAdminClient.create(),
+					SubscriptionAdminClient.create());
```

Comment:
???
I asked before security.
How all these `.create()` understand with what GC user they work?

Why this default, auto-configured `PubsubAdmin` doesn't depend on the `CredentialsProvider`?

That was really my first concern when I saw the same void `create()` in the `PubsubAdmin` ctors before.

Thanks